in-adapter-api/src/main/java/gohigher/application/ApplicationQueryController.java

```diff
@@ -4,27 +4,39 @@
 
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 import gohigher.application.port.in.ApplicationQueryPort;
-import gohigher.application.port.in.CalenderApplicationRequest;
-import gohigher.application.port.in.CalenderApplicationResponse;
+import gohigher.application.port.in.CalendarApplicationRequest;
+import gohigher.application.port.in.CalendarApplicationResponse;
+import gohigher.application.port.in.DateApplicationRequest;
+import gohigher.application.port.in.DateApplicationResponse;
 import gohigher.auth.support.Login;
 import gohigher.controller.response.GohigherResponse;
 import lombok.RequiredArgsConstructor;
 
 @RestController
 @RequiredArgsConstructor
+@RequestMapping("/v1/applications")
 public class ApplicationQueryController implements ApplicationQueryControllerDocs {
 
 	private final ApplicationQueryPort applicationQueryPort;
 
-	@GetMapping("/v1/applications/calender")
-	public ResponseEntity<GohigherResponse<List<CalenderApplicationResponse>>> findByMonth(@Login Long userId,
+	@GetMapping("/calendar")
+	public ResponseEntity<GohigherResponse<List<CalendarApplicationResponse>>> findByMonth(@Login Long userId,
 		@RequestParam int year, @RequestParam int month) {
-		CalenderApplicationRequest request = new CalenderApplicationRequest(userId, year, month);
-		List<CalenderApplicationResponse> response = applicationQueryPort.findByMonth(request);
+		CalendarApplicationRequest request = new CalendarApplicationRequest(userId, year, month);
+		List<CalendarApplicationResponse> response = applicationQueryPort.findByMonth(request);
+		return ResponseEntity.ok(GohigherResponse.success(response));
+	}
+
+	@GetMapping("/processes/date")
+	public ResponseEntity<GohigherResponse<List<DateApplicationResponse>>> findByDate(@Login Long userId,
+		@RequestParam String date) {
+		DateApplicationRequest dateApplicationRequest = new DateApplicationRequest(userId, date);
+		List<DateApplicationResponse> response = applicationQueryPort.findByDate(dateApplicationRequest);
 		return ResponseEntity.ok(GohigherResponse.success(response));
```

Comment:
```suggestion
		@ModelAttribute DateApplicationRequest request) {
		List<DateApplicationResponse> response = applicationQueryPort.findByDate(userId, request);
		return ResponseEntity.ok(GohigherResponse.success(response));
```

저번에 성원님 PR 에도 남겼던 리뷰인데, `@ModelAttribute` 가 아닌 `@RequestParam` 을 사용하신 이유가 있나요?

Reply:
큰 이유는 없지만 파라미터의 갯수가 3개 이하이기에 가독성 측면에서 파라미터가 낫다고 판단했습니다!